js/start.md

```diff
@@ -275,454 +229,570 @@ To install React-specific Amplify UI components, run the following command:
-    render() {
-      return (
-        <View style={styles.container}>
-          <Text>Welcome to your React Native App with Amplify!</Text>
-          <Button title="Generate Analytics Event" onPress={this.handleAnalyticsClick} />
-          {this.state.resultHtml}
-        </View>
-      );
-    }
+  return (
+    <View style={styles.container}>
+      <Button onPress={createNewTodo} title='Create Todo' />
+      { state.todos.map((todo, i) => <Text key={todo.id}>{todo.name} : {todo.description}</Text>) }
+    </View>
+  );
 }
+```
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-  link: {
-    color: 'blue'
-  }
-});
+Now if you wish to subscribe to data, import the `onCreateTodo` subscription and create a new subscription by adding subcription with `API.graphql()` like so:
+
+```javascript
+// other imports
+import { onCreateTodo } from './src/graphql/subscriptions'
+
+useEffect(() => {
+  //...other code
+
+  const subscription = API.graphql(graphqlOperation(onCreateTodo)).subscribe({
+      next: (eventData) => {
+        const todo = eventData.value.data.onCreateTodo;
+        dispatch({type:'SUBSCRIPTION', todo})
+      }
+  })
+  return () => subscription.unsubscribe()
+}, [])
 ```
 
-> The code above imports only the Auth and Analytics categories. To import the entire Amplify library use `import Amplify from 'aws-amplify'`. However, importing only the required categories is recommended as it will greatly reduce the final bundle size.
+The code above imports only the API and PubSub category. To import the entire Amplify library use `import Amplify from 'aws-amplify'`. However, importing only the required categories is recommended as it will greatly reduce the final bundle size.
+{: .callout .callout--info}
+
+After restarting your app using `npm start`, go back to `localhost:3000` in your browser and using dev tools you will see data being stored and retrieved in your backend from the console logs. At any time you can open the AWS console for your new API directly by running the following command:
+
+```terminal
+$ amplify console api
+> GraphQL               ##Select GraphQL
+```
+
+This will open the AWS AppSync console for you to run Queries, Mutations, or Subscriptions at the server and see the changes in your client app.
+
 
 </div>
 <div id="angular" class="tab-content">
 
-After creating your backend a configuration file will be generated in your configured source directory you identified in the `amplify init` command.
-
-Import the configuration file and load it in `main.ts`: 
+Update your `main.ts` to configure the library with `Amplify.configure()`:
 
 ```javascript
-import Amplify from 'aws-amplify';
-import amplify from './aws-exports';
-Amplify.configure(amplify);
+import PubSub from '@aws-amplify/pubsub';
+import API from '@aws-amplify/api';
+import awsconfig from './aws-exports';
+
+API.configure(awsconfig);
```

Comment:
When I followed these steps I got a `No pubsub module applied for subscription` error.  I think because we are just doing API.configure at this point?

Reply:
Was that just on Vue? And did you try stopping/restarting it? You shouldn’t need to do PubSub.configure() but I’ve noticed that the Vue hot module reloading sometimes requires the CLI to re-run